com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -546,40 +551,97 @@ public int additionalLightsShadowmapResolution
             get { return (int)m_AdditionalLightsShadowmapResolution; }
         }
 
+        /// <summary>
+        /// Controls the maximum distance at which shadows are visible.
+        /// </summary>
         public float shadowDistance
         {
             get { return m_ShadowDistance; }
             set { m_ShadowDistance = Mathf.Max(0.0f, value); }
         }
 
+        // MTT this is obsolete. What todo?
```

Comment:
I would make this class a partial class, create another partial version of it in Deprecated.cs, move this property there and mark it as [Obsolete("This is obsolete. Please use cascadeShadowSplitCount instead.", false).

Then when we get to 11.x we change the `false` to `true` and in 12.x we remove the property.

Reply:
And do the same for `m_ShadowCascades`

Reply:
Done.